blocks/rich-text/index.js

```diff
@@ -387,6 +400,19 @@ export class RichText extends Component {
 		this.context.onCreateUndoLevel();
 	}
 
+	getContainerNode() {
```

Comment:
I removed this function in https://github.com/WordPress/gutenberg/pull/5794/commits/45221727c4a4283637968792f6a38700697f1a9f since we are using a `ref` for the container now. Thanks for making PR https://github.com/WordPress/gutenberg/pull/6518 to help that happen!